modules/es-extensions/publisher/asset/policy/asset.js

```diff
@@ -211,6 +211,22 @@ asset.manager = function(ctx) {
             }
             asset.attributes["overview_version"] = asset.attributes["version"];
             return asset.attributes["version"];
+        },
+        getAssetGroup:function(asset){
+            var results = this._super.getAssetGroup.call(this,asset);
+            for (var index = 0; index < results.length; index++) {
+                var result = results[index];
+                var path = result.path;
+                var subPaths = path.split('/');
+                var name = subPaths[subPaths.length - 1];
```

Comment:
This we need not check since the path will not be null. So the length of the subPaths will at least be 1.